src/test/java/teammates/test/driver/AssertHelper.java

```diff
@@ -16,7 +16,19 @@
 import teammates.common.util.TimeHelper;
 
 public class AssertHelper {
-
+    
+    /**
+     * Assert date is now +- 1 min
+     */
+    public static void assertDateIsNow(Date date) {
+        assertDateWithinRange(date, new Date(new Date().getTime() - (1000 * 60)),
+                                    new Date(new Date().getTime() + (1000 * 60)));
```

Comment:
See if there is a utility method (in a helper class) for getting dates/times with an offset.